app/components/bali/rate/component.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module Bali
+  module Rate
+    class Component < ApplicationViewComponent
+      attr_reader :form, :method, :value, :scale, :size, :auto_submit, :options
+
+      def initialize(form:, method:, value:, scale: 1..5, auto_submit: false, **options)
+        @form = form
+        @method = method
+        @value = value
+        @scale = scale
+        @auto_submit = auto_submit
+        @size = options.delete(:size) || :medium
+
+        @options = prepend_class_name(options, 'rate-component')
+        @options = prepend_controller(options, 'rate')
```

Comment:
```suggestion
        @options = prepend_controller(@options, 'rate')
```
Si no mandas el @options en el prepend_controlloer, no estas sobrescirbiendo el `@options` ?

Reply:
si, buen punto